groups/groups.yaml

```diff
@@ -245,6 +245,7 @@ groups:
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     members:
+      - k8s-infra-release-admins@kubernetes.io
```

Comment:
This is related to https://github.com/kubernetes/k8s.io/issues/157#issuecomment-718043478:

> Ensure that the community (someone who isn't @listx or @thockin) is able to respond to an alert, possibly proven via some kind of DiRT (Disaster Recovery Training) scenario

So I believe these would be alerts coming directly from GCP, not Testgrid (since the auditor runs from Cloud Run).

Staring at https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/Vanity-Domain-Flip.md#alerts:

> The Stackdriver Error Reporting leg of the auditing process is responsible for sending alerts to humans about the rejected GCR change.
>
> Currently, an email is sent to the project owner(s) of the k8s-artifacts-prod GCP project.

Implies that we might not have an appropriately-scoped group today to send auditor alerts to.